.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Should this file be included or should the `/.vscode/` folder be excluded?

Reply:
I think it's generally good practice to include the .vscode folder so everyone working on the project has the same setup